README.md

```diff
@@ -73,54 +154,69 @@ USE_L10N = True
 ```
 
 A list of supported language codes is available [here](https://www.transifex.com/explore/languages/) and should
 be declared in the `ll-cc` format, compatible with the `Accept-Language` HTTP header specification, for example
 `pt-br` instead of `pt_BR`.
 
-### Quick guide
+#### Quickstart
 
 These are the minimum steps required for testing the Transifex Toolkit with a Django project end-to-end:
 
-1. Add translation hooks in your templates
-2. Push the source content to Transifex
-3. Translate content on Transifex
-4. Display translated content to your users
+1. Install Transifex Python toolkit in your Django project, using the pip install transifex-python in your console. Read more about [toolkit installation](#installation).
+2. Update your Django settings file with instruction on using the Transifecx Native toolkit. Read more about [configuring your project](#configuration).
+3. [Add translation hooks in your templates](#add-translation-hooks)
+4. [Push the source content to Transifex](#push-source-content-to-transifex)
+5. [Translate content on Transifex](#translate-content-on-transifex)
+6. [Display translated content to your users](#display-translated-content)
+
+##### Add Translation Hooks
 
-#### 1. Add translation hooks
+###### Django template simple example 
 
 Open a Django template file (e.g. an `.html` file) and add the following:
-```
-{% load transifex %}
 
+```python
+{% load transifex %}
 <p>{% t "Hello!" %}</p>
 <p>{% t "I want to be translated." %}</p>
 ```
 
-#### 2. Push source content to Transifex
+###### Django view simple example 
 
-This command will collect all translatable strings and push them to Transifex.
+```python
+from transifex.native.django import t
+from django.http import HttpResponse
+
+def my_view(request):
+    output = t("Welcome aboard!")
+    return HttpResponse(output)
 ```
+	
+To see more examples on translation hooks checkout [Use in Templates and Views](#use-in-templates-and-views).	
+
+##### Push Source Content to Transifex
+
+This command will collect all translatable strings and push them to Transifex.
+
+```bash
 ./manage.py transifex push
 ```
 
-#### 3. Translate content on Transifex
+##### Translate Content on Transifex
 
 The next step is for your translators to translate the strings in various languages using Transifex.
-When a translation is added on Transifex, it becomes available over-the-air on your app.
-Please note that it can take a few minutes for the translations to become available on your app.
+When a translation is added on Transifex, it becomes available over-the-air on your application.
+Please note that it can take a few minutes for the translations to become available on your application.
 
-#### 4. Display translated content
+##### Display Translated Content
 
 The Transifex Toolkit automatically displays translated content in the language currently selected in your Django project.
+In order to allow changing the current language, you will need a **language picker**.
```

Comment:
A suggestion. When I tried to set things up for a demo project, I forgot the change in the URLs and the change in the middleware. Perhaps this would be better as a numbered list:

```
1. You can add this on the same HTML file you added the translatable strings before, like so:

   \```html
   <form>
   <!-- ... --!>
   \```

2. Add the following route in your Project's routes, so that the `set_language` hook shown above will work when submitting the form.

   \```python
   from django.conf.urls import url, include
   # ...
   \```

3. Last, add `'django.middleware.locale.LocaleMiddleware'` in your `settings.MIDDLEWARE` to enable the functionality.
```

Also, maybe a link for the relevant part of django's documentation:
- https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#the-set-language-redirect-view
- https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#how-django-discovers-language-preference

Reply:
Done, nice comment!

Reply:
Uhm, I removed parts of the language picker form so that my comment wouldn't be very large. I think the full language picker example should be visible in the readme.

Reply:
Oh, actually, the full snippet for the language picker is repeated.

Reply:
Correct, I didn't understand your approach, now I think it's ok :)